src/components/TabbedNav/TabbedNav.jsx

```diff
@@ -0,0 +1,42 @@
+const React = require('react')
+const hookUpStickyNav = require('../hookUpStickyNav')
+
+class TabbedNav extends React.Component {
+  render() {
+    return (
+      <nav id="tabbed-nav" className="top-nav nav tabs is-toggle is-fullwidth">
+        <ul>
+          <li className={this.props.active === 'tasks' ? 'is-active' : ''}>
+            <a id="notifications-link" onClick={this.props.showTasks}>
+              <span className="tab octicon octicon-mail"></span>
+              <span>Tasks</span>
+            </a>
+          </li>
+          <li className={this.props.active === 'filters' ? 'is-active' : ''}>
+            <a id="filters-link" onClick={this.props.manageFilters}>
+              <span className="tab octicon octicon-beaker"></span><span>Filters</span>
+            </a>
+          </li>
+          <li className={this.props.active === 'auth' ? 'is-active' : ''}>
+            {typeof this.props.user === 'object' ? (
```

Comment:
Could this `typeof` check be moved outside the `<li` so that the list item is only included if we do have a user? That might screw up the appearance. Just thinking we don't need an empty li sitting around.